backend/src/main/java/com/engineering/thesis/backend/repository/UserRepository.java

```diff
@@ -7,6 +7,8 @@
 
 public interface UserRepository extends JpaRepository<User, Long> {
     Boolean existsByEmailAndIsActive(String email, boolean isActive);
+
     Optional<User> findAllByEmail(String email);
+
```

Comment:
W interface'ach usuwam specjalnie entery, bo są niepotrzebne. Im mniej enterów tym łatwiej przeklikiwać w nich metody bez przewijania. Jak to autokorekta robi to po prostu pousuwać potem entery ;)